roles/swift-container/handlers/main.yml

```diff
@@ -1,3 +1,11 @@
 ---
-- name: restart swift-container service
-  service: name=swift-container state=restarted
+- name: restart-swift-container-services
```

Comment:
This should be "restart swift-container-services" based on how the handlers called from the container task.